website/content/api-docs/admin-partitions.mdx

```diff
@@ -13,23 +13,18 @@ The functionality described here is available only in
 
 ## Create a Partition
 
-This endpoint creates a new Partition.
-
-| Method | Path         | Produces           |
-| ------ | ------------ | ------------------ |
-| `PUT`  | `/partition` | `application/json` |
-
-The table below shows this endpoint's support for
-[blocking queries](/api/features/blocking),
-[consistency modes](/api/features/consistency),
-[agent caching](/api/features/caching), and
-[required ACLs](/api#authentication).
-
-| Blocking Queries | Consistency Modes | Agent Caching | ACL Required     |
-| ---------------- | ----------------- | ------------- | ---------------- |
-| `NO`             | `none`            | `none`        | `operator:write` |
-
-The corresponding CLI command is [`consul partition create`](/commands/partition#create).
+This endpoint creates a new Partition and has the following characteristics:
+
+| Characteristic | Value |
+| :------------- | :---- |
```

Comment:
The colon forces left-aligned header text

Reply:
Yes, we will maintain the flexibility to change alignment. We just want to make the default alignment be correct most of the time (left-aligned). This is captured in more detail in Asana: https://app.asana.com/0/1100423001970639/1201999608008890/f